### ISSUE TITLE
ESP32-RFID Project with WebServer version for ESP8266

### ISSUE THREAD

**Issue by FrightRisk** [donskytech/platformio-projects#3]
It looks like you have an IFDEF to have this project run on an ESP8266, could you help provide some tips about making this run on an ESP8266/ESP8285? I have some DoIt MX Devboards and a lot of ESP-01s boards. I don't know if it can run on those or not, but I would like to at least get it to run on the dev boards. I changed my platformio.ini to this:

[env:esp32dev]
platform = espressif32
board = esp32dev
framework = arduino
monitor_speed = 115200
lib_deps = 
	https://github.com/OSSLibraries/Arduino_MFRC522v2.git
	Wire
	Wifi
	FS
	SPIFFS
	https://github.com/me-no-dev/ESPAsyncWebServer.git
	bblanchon/ArduinoJson@^6.19.4

[env:espmxdevkit]
platform = espressif8266
board = espmxdevkit
framework = arduino
monitor_speed = 115200
lib_deps = 
	https://github.com/OSSLibraries/Arduino_MFRC522v2.git
	Wire
	ESP8266WiFi
	FS
	SPIFFS
	me-no-dev/ESP Async WebServer@^1.2.3
	bblanchon/ArduinoJson@^6.19.4

It will allow compiling for the ESP32, but not for the 8266. I get the following warnings and errors:

Library Manager: Installing FS
Warning! Could not find the package with 'FS' requirements for your system 'windows_amd64'
Library Manager: Installing SPIFFS
Warning! Could not find the package with 'SPIFFS' requirements for your system 'windows_amd64'
src\main.cpp:116:8: warning: 'SPIFFS' is deprecated: SPIFFS has been deprecated. Please consider moving to LittleFS or other filesystems. [-Wdeprecated-declarations]
src\main.cpp:116:25: error: no matching function for call to 'fs::FS::begin(bool)'

SPIFFS is deprecated, so it would probably be a good idea to replace it with LittleFS, but LittleFS has a minimum of 4k allocation unit per file while SPIFFS had only 256 bytes. I don't know if the way you are using the file system would fit on an ESP-01s yet, but it would be pretty cool to have this run on a fingernail sized board!

I appreciate any help you can provide, even to just point me in the right direction about how to fix the issues. Thank you!

**Comment by donskytech** [comment]
Yes I think for your ESP8266 remove all SPIFFS and replace it with LittleFS.
Maybe this could help you https://www.donskytech.com/esp8266-webserver-using-littlefs/#htoc-esp8266-webserver-no-littlefs-spiffs

```
[env:nodemcuv2]
platform = espressif8266
board = nodemcuv2
framework = arduino
board_build.filesystem = littlefs
lib_deps = ottowinter/ESPAsyncWebServer-esphome@^3.0.0
; Serial Monitor options
monitor_speed = 115200
```
For ESP-01, I have not tried actually.
